src/containers/Home/styles.module.css

```diff
@@ -15,3 +16,12 @@
 .dropdownContainer{
   margin: auto 30px
 }
+
+.sourceFromDateContainer{
+  margin: auto 30px;
+  margin-left: 135px;
```

Comment:
We may need to change the approach to style this container to get rid of some issues. Let's discuss this on a call.

Reply:
Ok, I'll be waiting!